src/index.html

```diff
@@ -1,14 +1,67 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <header class="header">
+      <a class="icon-ref" href="">
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+          class="icon"
+        >
+      </a>
+      <nav >
+        <ul class="menu">
+          <li>
+            <a href="#apple" class="link is-active">apple</a>
+          </li>
+          <li>
+            <a href="#samsung" class="link ">samsung</a>
```

Comment:
видаляй зайві пробіли
```suggestion
            <a href="#samsung" class="link">samsung</a>
```

Reply:
понял, пасибо!